mParticle-Apple-SDK/MPRokt.m

```diff
@@ -120,6 +120,18 @@ - (void)events:(NSString * _Nonnull)identifier onEvent:(void (^ _Nullable)(MPRok
     });
 }
 
+- (void)close {
+    dispatch_async(dispatch_get_main_queue(), ^{
+        // Forwarding call to kits
+        [[MParticle sharedInstance].kitContainer_PRIVATE forwardSDKCall:@selector(close)
```

Comment:
Generally, any method called on the kits this way has been added to the KitProtocol first. But the Rokt SDK is a pretty obvious exception compared to the rest of our kits so I agree this makes sense.